yoti_python_sdk/tests/dynamic_sharing_service/test_dynamic_scenario_builder.py

```diff
@@ -31,3 +31,14 @@ def test_build_scenario():
     assert EXTENSION1 in scenario["extensions"]
     assert EXTENSION2 in scenario["extensions"]
     assert scenario["callback_endpoint"] == CALLBACK_ENDPOINT
+
+
+def test_auto_allow():
```

Comment:
If we're going to remove withAutoAllow from the source constraint, we won't need this test anymore